go.sum

```diff
@@ -1,5 +1,11 @@
+github.com/alecthomas/units v0.0.0-20211218093645-b94a6e3cc137 h1:s6gZFSlWYmbqAuRjVTiNNhvNRfY2Wxp9nhfyel4rklc=
```

Comment:
Same comments as in the go.mod

Reply:
i think this showed up after go mod tidy

Reply:
after removing it 

I get this when I run 

```
ekyale@yow-ekyale-lx:~/go/src/github.com/gophercloud/gophercloud(story/lowlatency_updates)$ go vet ./starlingx/...
starlingx/inventory/v1/osds/requests.go:7:2: no required module provides package github.com/alecthomas/units; to add it:
        go get github.com/alecthomas/units
```

Reply:
go fmt ./starlingx/... also tries to format several files 

```
ekyale@yow-ekyale-lx:~/go/src/github.com/gophercloud/gophercloud(story/lowlatency_updates)$ go fmt ./starlingx/...
starlingx/inventory/v1/osds/requests.go
starlingx/inventory/v1/partitions/requests.go
starlingx/inventory/v1/ptpinstances/requests.go
starlingx/inventory/v1/ptpinstances/testing/fixtures.go
starlingx/inventory/v1/ptpinstances/testing/requests_test.go
starlingx/inventory/v1/ptpinterfaces/requests.go
starlingx/inventory/v1/ptpinterfaces/testing/fixtures.go
starlingx/inventory/v1/ptpinterfaces/testing/requests_test.go
starlingx/inventory/v1/ptpparameters/results.go
starlingx/inventory/v1/ptpparameters/testing/fixtures.go
starlingx/inventory/v1/ptpparameters/testing/requests_test.go
starlingx/inventory/v1/serviceparameters/testing/fixtures.go
```